src/test/scala/uk/gov/hmrc/api/specs/AuthorisationsSpec.scala

```diff
@@ -298,8 +298,8 @@ class AuthorisationsSpec extends BaseSpec with EoriGenerator with TestData {
     }
   }
 
-  Feature("500, INTERNAL_SERVER_ERROR case Scenarios") { // TODO: A different way to generate a 500 from the API/Service?
```

Comment:
I've removed the 500 all together, as I am unsure of how we can generate one now?